_common/qiskit/execute.py

```diff
@@ -149,10 +149,9 @@ def __init__(self, qiskit_result):
     def get_counts(self, qc=0):
         # counts= self.qiskit_result.quasi_dists[0].binary_probabilities()
         # for key in counts.keys():
-        #     counts[key] = int(counts[key] * self.qiskit_result.metadata[0]['shots'])        
+        #     counts[key] = int(counts[key] * self.qiskit_result.metadata[0]['shots'])
         qc_index = 0 # this should point to the index of the circuit in a pub
-        register_name = list(self.qiskit_result[qc_index].data.keys())[0]
-        bitvals = getattr(self.qiskit_result[qc_index].data, register_name)
+        bitvals = next(iter(self.qiskit_result[qc_index].data.values()))
```

Comment:
If you are interested in only the first register, you can directly get the result like this line.
Because `DataBin` becomes a mapping since 1.1.0, you don't have to use `getattr` anymore.
You can access data by `result[i].data["register_name"]`.